tests/autoyast/installation.pm

```diff
@@ -111,6 +111,8 @@ sub run {
             last if match_has_tag("bios-boot") || match_has_tag("reboot-after-installation");
 
             if (match_has_tag('autoyast-error')) {
+                my $reason = get_var('AUTOYAST_EXPECT_ERRORS_REASON');
+                record_soft_failure "AUTOYAST_EXPECT_ERRORS $reason";
```

Comment:
I think `record_soft_failure 'AUTOYAST_EXPECT_ERRORS ' . get_var('AUTOYAST_EXPECT_ERRORS_REASON');` should be enough